src/hott/06-contractible.rzk.md

```diff
@@ -240,23 +242,24 @@ A retract of contractible types is contractible.
         ( is-retract-of-retraction)
         ( contracting-htpy B is-contr-B (is-retract-of-section a)))
       ( is-retract-of-homotopy a)
+```
 
--- If A is a retract of a contractible type it is contractible.
-#def is-retract-of-is-contr-is-contr uses (AretractB)
+```rzk title="If $A$ is a retract of a contractible type it is contractible"
```

Comment:
Same as above:

```suggestion
```rzk title="If A is a retract of a contractible type it is contractible"
```